### PR TITLE
Solution #3126

### PR DESCRIPTION
**Pull request by PavelAplevich**
**@misaochan Sorry for the delay**
Result of some locale

**Russian:**
https://commons.wikimedia.beta.wmflabs.org/w/index.php?title=File:Kran5.jpg&action=edit&section=1

**Eng(UK):**
https://commons.wikimedia.beta.wmflabs.org/w/index.php?title=File:Kran6.jpg&action=edit&section=1
https://commons.wikimedia.beta.wmflabs.org/w/index.php?title=File:PoolEng.jpg&action=edit&section=1

**CzechRepublic:**
https://commons.wikimedia.beta.wmflabs.org/w/index.php?title=File:CranCzech.jpg&action=edit&section=1
https://commons.wikimedia.beta.wmflabs.org/w/index.php?title=File:PoolCzech.jpg&action=edit&section=1

**CzechRepublic noEXIF:**
https://commons.wikimedia.beta.wmflabs.org/w/index.php?title=File:NoEXIF.jpg&action=edit&section=1

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java [hunk 1]
```diff
@@ -202,15 +202,11 @@ public String getPageContents(Context applicationContext) {
     private String getTemplatizedCreatedDate() {
         if (dateCreated != null) {
             if (UploadableFile.DateTimeWithSource.EXIF_SOURCE.equals(dateCreatedSource)) {
-                return String.format(Locale.ENGLISH, TEMPLATE_DATE_ACC_TO_EXIF, DateUtil.getDateStringWithSkeletonPattern(dateCreated, "yyyy-MM-dd")) + "\n";
+                java.text.SimpleDateFormat dateformat = new java.text.SimpleDateFormat("yyyy-MM-dd");
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java]: Please rename the variable to `dateFormat`, thanks! 🙂

**Comment by nicolas-raoul** [comment]
The results look great! After the variable name change I would say this can be merged.

**Comment by PavelAplevich** [comment]
@nicolas-raoul All done

File: app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java [hunk 2]
```diff
@@ -202,15 +202,11 @@ public String getPageContents(Context applicationContext) {
...
+                return String.format(Locale.ENGLISH, TEMPLATE_DATE_ACC_TO_EXIF, dateFormat.format(dateCreated)) + "\n";
             } else {
-                Calendar calendar = Calendar.getInstance();
-                calendar.setTime(dateCreated);
-                calendar.setTimeZone(TimeZone.getTimeZone("UTC"));
-                return String.format(Locale.ENGLISH, TEMPLATE_DATA_OTHER_SOURCE,
-                        calendar.get(Calendar.YEAR),
-                        calendar.get(Calendar.MONTH),
-                        calendar.get(Calendar.DAY_OF_MONTH)) + "\n";
+                java.text.SimpleDateFormat dateFormat = new java.text.SimpleDateFormat("yyyy-MM-dd");
```
Comment (ashishkumar468) [inline app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java]: dateFormat could be placed out of if-else, the same thing is used in both if and else

**Comment by ashishkumar468** [comment]
Thanks for the PR @PavelAplevich . Everthing works fine for me. Happy to merge after the comment is addressed :)

**Comment by PavelAplevich** [comment]
@ashishkumar468 I hope that's all my fails)

File: app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java (hunk 1 above)
Comment (ashishkumar468) [reply app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java]: @PavelAplevich  Please rename dateformat to dateFormat :)

Comment (PavelAplevich) [reply app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java]: @ashishkumar468 All done already, see last commit

**Pull request merged**